ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+    ${return_code}    ${podname}    Run And Return Rc And Output  oc get pods -n ${PRJ_TITLE} -o json | jq '.items[].metadata.name' | grep 'modelmesh-serving'
+    Log    Serving Runtime Podname: ${podname}
+    ${MM_PAYLOAD_PROCESSORS_Actual} =  Run  oc get pod ${podname} -n ${PRJ_TITLE} -o json | jq '.spec.containers[0].env[] | select(.name=="MM_PAYLOAD_PROCESSORS") | .value'
+    ${MM_PAYLOAD_PROCESSORS_Actual} =    Strip String    ${MM_PAYLOAD_PROCESSORS_Actual}    characters="
+    Should Be Equal  ${MM_PAYLOAD_PROCESSORS_Actual.strip()}  ${MM_PAYLOAD_PROCESSORS_Expected.strip()}
+
+Verify One Model Serving Pod Exists
+    [Documentation]    Verifies that serving runtime pods have been stabilized and only 1 pod exists
+    [Arguments]    ${PRJ_TITLE}     ${label_selector}
+    ${return_code}    ${pod_number}    Run And Return Rc And Output   oc get pod -n ${PRJ_TITLE} -l ${label_selector} | tail -n +2 | wc -l
+    ${pod_numbers}    Split String     ${pod_number}
+    Should Be Equal    ${pod_numbers}[0]   1
+
+#Verify MM_PAYLOAD_PROCESSORS Environment Variable Is Removed
+#    [Documentation]    Verify that the MM_PAYLOAD_PROCESSORS is deleted when TrustyAI service is removed
+#    ${podname} =  Run  oc get pods -n ${PRJ_TITLE} -o json | jq '.items[].metadata.name' | grep 'modelmesh-serving'
+#
+
+Send Batch Inference Data to Model
+    [Documentation]    Send Batch Inference data to the already deployed model using Curl commands
+    [Arguments]        ${model_name}   ${project_name}
+    ${url}=    Get Model Route via CLI    ${model_name}   ${project_name}
+    #To be removed PWD Command
+    ${rc}    ${pwd}=    Run And Return Rc And Output    echo $PWD
+    FOR    ${counter}    IN RANGE    1    5
+        ${inference}=  Set Variable   ${pwd}/ods_ci/tests/Resources/Files/TrustyAI/loan_default_batched/batch_0${counter}.json
+        ${curl_cmd}=     Set Variable    curl -sk ${url} -d @${inference}
+        ${rc}  ${inference_output}=    Run And Return Rc And Output    ${curl_cmd}
+        Should Contain    ${inference_output}    model_name
+        Should Be Equal As Integers	${rc}	 0
+    END
+
+Get ModelId For A Deployed Model
+    [Documentation]   Curl command to get modelid of the model. The sufffix gets changed by modelmesh
+    ...               https://github.com/trustyai-explainability/trustyai-explainability/issues/395
+    [Arguments]       ${model_name}
+    #To Be removed later once modelid doesn't get modified
+    ${curl_cmd}=     Set Variable    curl -sk --location ${TRUSTY_ROUTE}/info | jq '.[0].data.modelId'
+    ${rc}  ${output}=     Run And Return Rc And Output    ${curl_cmd}
+    ${model_id}=    Set Variable If    '${output}'=='${EMPTY}'    ${model_name}    ${output}
+    RETURN    ${model_id}
+
+Create Metrics request via CLI
+    [Documentation]    Create an SPD or DIR metrics via CLI
```

Comment:
you mean to say query the metrics?

Reply:
Is is actually scheduling a request: [OPERATOR_NAMESPACE](https://github.com/trustyai-explainability/odh-trustyai-demos/tree/main/2-BiasMonitoring#schedule-a-fairness-metric-request) Can rename if needed.

Reply:
oh, okay. And don't we need to wait for the request result to be computed?

Reply:
No, it doesn't need a wait time. Updating the name (schedule a bias metric request) to match  syntax here https://github.com/trustyai-explainability/odh-trustyai-demos/blob/main/2-BiasMonitoring/README.md#schedule-a-fairness-metric-request

Reply:
ok sounds good. Pls update the documentation accordingly